sceptre/nlpsandbox/config/common/peer-nlpsandbox-vpc-admincentral.yaml

```diff
@@ -7,4 +7,4 @@ parameters:
   VpnCidr: 10.1.0.0/16
 hooks:
   before_launch:
-    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/master/peer-route-config.yaml --create-dirs -o templates/remote/peer-route-config.yaml"
+    - !cmd "wget {{stack_group_config.aws_infra_templates_root_url}}/v0.2.5/peer-route-config.yaml -N -P templates/remote"
```

Comment:
the reference you have is `https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.5/peer-route-config.yaml`  which is not correct.  what you want is.. 
 `https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.5/templates/peer-route-config.yaml`